### PR TITLE
fix SIR example

### PR DESCRIPTION
**Pull request by rakvat**
This fixes the SIR example for me. 

Might fix https://github.com/JuliaDynamics/Agents.jl/issues/1026

### CODE REVIEW

**Comment by Tortar** [comment]
thanks a lot!! We actually track the step of the simulation through `abmtime` so I update your fix with it :-)

**Comment by Tortar** [comment]
Let's see if on the online docs everything works fine now after it builds

File: examples/sir.jl [hunk 1]
```diff
@@ -94,10 +93,10 @@ function model_initiation(;
         reinfection_probability,
         detection_time,
         C,
-        death_rate
+        death_rate,
     )
     space = GraphSpace(complete_graph(C))
-    model = StandardABM(PoorSoul, space; agent_step!, properties, rng)
+    model = StandardABM(PoorSoul, space; agent_step!, model_step!, properties, rng)
```
Comment (rakvat) [inline examples/sir.jl:99]: can't you remove this change as well now?

Comment (Tortar) [reply examples/sir.jl:99]: sure, I assumed there was one before, that's what happens when you are multitasking xD

**Review by Tortar — APPROVED** [review]

**Pull request merged**